main.py

```diff
@@ -44,6 +44,9 @@ def main():
             with open("config.json", 'w') as file:
                 json.dump(configuration, file, sort_keys=True, indent=4, ensure_ascii=False)
 
+        player_max = configuration["player_max"]
+        player_online = configuration["player_online"]
+        protocol = configuration["protocol"]
```

Comment:
I would think a default protocol of `2` would be good here, since this server is intended to be used when a maintenance is ongoing